package.json

```diff
@@ -279,6 +280,9 @@
     "graphql": "16.8.0"
   },
   "version": "0.2.1",
+  "scripts": {
+    "start": "cross-env FORCE_COLOR=true concurrently -n \"TwentyServer,TwentyFrontend\" -c \"bgBlue.bold,bgGreen.bold\" \"yarn nx start:dev twenty-server\" \"yarn nx start twenty-front\""
```

Comment:
Can we replace "TwentyServer,TwentyFrontend" by "twenty-server,twenty-front" to be consistent folder cases.

Let's give it a week to try it on various environments and then we can add it to the docs
Also, could you merge main into your branch, tests should be green!

Reply:
Hi @charlesBochet 
Sorry I just saw this I was bit busy